lessons/basics/strings.md

```diff
@@ -12,11 +12,12 @@ What are Strings in Elixir, Char lists, Graphemes and Codepoints.
 
 - [Strings in Elixir](#strings-in-elixir)
 - [Char lists](#char-lists)
-- [Graphemes and Codepoints](#graphemes)
+- [Graphemes and Codepoints](#graphemes-and-codepoints)
 - [String functions](#string-functions)
-  - [Length](#length)
-  - [Replace](#replace)
-  - [Duplicate](#duplicate)
+  - [Length](#length1)
```

Comment:
@jeanmatheussouto instead of changing these, how about we update the headers to no longer be code instead?

For instance, changing line 72 from `##### length/1` to `##### Length`.